Dockerfile

```diff
@@ -70,16 +70,12 @@ RUN VERSION=v0.7.5 &&\
     wget -qO- https://github.com/containernetworking/plugins/releases/download/$VERSION/cni-plugins-amd64-$VERSION.tgz \
         | tar xfz - -C /opt/cni/bin
 
-# Install crictl
-ENV CRICTL_COMMIT ff8d2e81baf8ff720fb916e42da57c2b772bd19e
-RUN set -x \
-       && export GOPATH="$(mktemp -d)" \
-       && git clone https://github.com/kubernetes-sigs/cri-tools.git "$GOPATH/src/github.com/kubernetes-sigs/cri-tools" \
-       && cd "$GOPATH/src/github.com/kubernetes-sigs/cri-tools" \
-       && git checkout -q "$CRICTL_COMMIT" \
-       && go install github.com/kubernetes-sigs/cri-tools/cmd/crictl \
-       && cp "$GOPATH"/bin/crictl /usr/bin/ \
-       && rm -rf "$GOPATH"
+# Install crictl and critest
+RUN VERSION=v1.14.0 &&\
```

Comment:
Is there a way to get the cri-o version and parse it, instead of setting it here? Adding one here will likely be missed when bumping releases

Reply:
Otherwise, a pointer in version/version.go should exist to make sure it's not missed

Reply:
Hm, we could vendor critest and crictl into the repo and only rely on that. Updating would be done via dependabot. WDYT?

Reply:
This should be resolved now since I do not touch the file any more. I will see If we can find another way to keep the Dockerfile dependencies up to date.